chain/client/src/client_actor.rs

```diff
@@ -756,7 +756,11 @@ impl Handler<Status> for ClientActor {
                 last_block_height = block.header().height();
             }
 
-            Some(DetailedDebugStatus { last_blocks: blocks_debug })
+            Some(DetailedDebugStatus {
+                last_blocks: blocks_debug,
+                network_info: self.network_info.clone().into(),
+                sync_status: self.client.sync_status.as_variant_name().to_string(),
```

Comment:
Sync status should display more information than just the enum itself. For example, in HeaderSync, it would be nice to know where the current header head is vs where it is trying to sync to. Please see `display_sync_status` for what it displayed there.

This can be done in a separate PR though